python/grass/temporal/abstract_space_time_dataset.py

```diff
@@ -2780,8 +2780,18 @@ def update_from_registered_maps(self, dbif=None):
 
         # Update the spatial and temporal extent from registered maps
         # Read the SQL template
+        template_suffix = ""
+        db_backend = dbif.get_dbmi().__name__
+        old_sqlite_version = (
+            db_backend == "sqlite3" and dbif.get_dbmi().sqlite_version_info < (3, 33)
+        )
```

Comment:
The `old` might not age very well. Maybe something more descriptive?

Reply:
I would agree, if this was meant to stay in the code for an unforeseeable future. But given that we intend to revert / remove this again in (I would suggest April 2025, when Ubuntu 20.04 is EOL), "old" might be OK?

Reply:
I think both of these are valid points, but I guess I'm more concerned with having a clear note about when to remove it. "Remove in April 2025 when Ubuntu 20.04 is EOL." If we have many of those, we could create an issue of each, create a GitHub project to track those, and dates can be then assigned to issues within a project. Possible overkill.

Reply:
I can open an issue "Revert #3723" when I merge the PR, no?
A comment in the code is probably not too helpful as changes are at different places...

Reply:
Good idea for the issue, please do